static/sass/_pattern_animations.scss

```diff
@@ -0,0 +1,55 @@
+@mixin canonical-u-animations {
+  .u-animation--slide-from-bottom {
+    opacity: 0;
+    animation: slide-from-bottom map-get($animation-duration, sleepy) linear
```

Comment:
missed this yesterday - `animation` should appear before `opacity` in these three classes